### PR TITLE
Proposal for CI for PRs submitted for cordova plugins

### PR DESCRIPTION
**Pull request by sarangan12**

### CODE REVIEW

**Comment by nikhilkh** [comment]
@purplecabbage @stevengill @csantanapr to help review and comment.

**Comment by shazron** [comment]
Thanks to Microsoft for sponsoring the CI machines on Azure :)
As usual please provide installation/deployment docs also just in case we need to move off Azure in the future, for whatever reason.

**Comment by nikhilkh** [comment]
@sarangan12 I believe you were planning to document how Jenkins CI is being configured. Can you provide more details?

File: proposals/ci-for-plugin-prs.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+## Background
+Currently, Cordova plugins have [continuous intergration builds][buildbot] running:
+
+* Periodically with master branch
+* Per commit
+
+This process, though helpful, have the following disadvantages:
+
+* Code issues, that might break cordova, are not caught before checking in.
+* There is no easy for the code submitter to reproduce the error locally
```
Comment (omefire) [inline proposals/ci-for-plugin-prs.md]: missing a word: 'way'.

Comment (sarangan12) [inline proposals/ci-for-plugin-prs.md]: Done

**Comment by sarangan12** [comment]
Instructions (if we need to move out of azure) is available [here](https://github.com/apache/cordova-medic/blob/master/JENKINS.md).

**Pull request merged**